autogpt_platform/backend/backend/data/credit.py

```diff
@@ -1023,3 +1020,26 @@ async def get_auto_top_up(user_id: str) -> AutoTopUpConfig:
         return AutoTopUpConfig(threshold=0, amount=0)
 
     return AutoTopUpConfig.model_validate(user.topUpConfig)
+
+
+async def get_graph_execution_cost(graph_exec_id: str) -> int | None:
```

Comment:
Cost is now already available in the `cost` field inside the `stats` column of the execution, we should just use that to avoid another nested db lookup https://github.com/Significant-Gravitas/AutoGPT/blob/52ee7d150e6bac2db028b4fcbe0aa94cd7362bfe/autogpt_platform/backend/backend/executor/manager.py#L747

Reply:
Thanks, that's a lot easier :)